### PR TITLE
Feat(MPM-533): rdkafka6.0-support

### PR DESCRIPTION
**Pull request by ruzakaranbol**

### CODE REVIEW

File: bkp_composer.lock [hunk 1]
```diff
@@ -0,0 +1,5501 @@
+{
```
Comment (nick-zh) [inline bkp_composer.lock]: Accidental commit? We used to not commit locks for libraries :v:

Comment (ruzakaranbol) [reply bkp_composer.lock]: It was added accidentally.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  ci-caching: jobcloud/ci-caching@1.0.2
-  ci-php: jobcloud/ci-php@0.32
+  ci-caching: jobcloud/ci-caching@3.0
```
Comment (bajdzun) [inline .circleci/config.yml:4]:
```suggestion
  ci-caching: jobcloud/ci-caching@4.0
```

Comment (bajdzun) [reply .circleci/config.yml:4]: Sorry, my bad, thought it was the other orb 😄

**Comment by SilverFire** [comment]
Hello.
It would be nice to merge this PR and issue a release, if the code is OK for you

**Review by Steveb-p — DISMISSED** [review]

**Review by bajdzun — APPROVED** [review]

**Comment by healerz** [comment]
@SilverFire we released the new version. We were waiting for the release of [this fix](https://github.com/arnaud-lb/php-rdkafka/pull/514).

**Pull request merged**